README.md

```diff
@@ -3,8 +3,11 @@
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/datadog)](https://artifacthub.io/packages/search?repo=datadog)
 
 Official Helm charts for Datadog products. Currently supported:
-- [Datadog Agents](charts/datadog/README.md) (datadog/datadog)
-- [Observability Pipelines Worker](charts/observability-pipelines-worker/README.md) (datadog/observability-pipelines-worker)
+- [Datadog Agents](charts/datadog/README.md) (`datadog/datadog`)
+- [Datadog Operator](charts/datadog-operator/README.md) (`datadog/datadog-operator`)
+- [Extended DaemonSet](charts/extended-daemon-set/README.md) (`datadog/extendeddaemonset`)
+- [Observability Pipelines Worker](charts/observability-pipelines-worker/README.md) (`datadog/observability-pipelines-worker`)
+- [Synthetics Private Location](charts/synthetics-private-location/README.md) (`datadog/synthetics-private-location`)
```

Comment:
Skipped [datadog-crds](https://github.com/DataDog/helm-charts/tree/main/charts/datadog-crds) as I believe it is not meant to be used on its own